docker/complement/conf-workers/postgres.supervisord.conf

```diff
@@ -0,0 +1,16 @@
+[program:postgres]
+command=/usr/local/bin/prefix-log /usr/bin/pg_ctlcluster 13 main start --foreground
+
+# Lower priority number = starts first
+priority=100
```

Comment:
following up on https://github.com/matrix-org/synapse/pull/12480#issuecomment-1110942061: 

> > Do we actually need postgres to be a higher prority than synapse? It's not like there's any guarantee that doing so will mean that postgres is ready before synapse starts.
> 
> No, though Synapse's configuration already defined it at priority `500` so it seemed like this was the intention.

I'm not sure the priority was particularly carefully chosen, tbh. (ftr, this bit came from #9162).

> (I just realised: The main Synapse process' priority is actually defined as `10`; Redis' is `1`. Postgres should probably be on the same level as Redis if we're going to have priority ordering?)

If we're going to have priority ordering, then yes it ought to be lower than Synapse. 

> I would have thought that Postgres being up first might avoid some noisy errors in Synapse's logs.

If it actually manages to start up in 1 second, then yes maybe. TBH though I would optimise for fastest runtime rather than lack of noise in the logs.

Reply:
let's set this to be the same as Redis for now, and get this PR merged.

Reply:
oh you're way ahead of me